pkg/resources/database.go

```diff
@@ -22,6 +25,11 @@ var databaseSchema = map[string]*schema.Schema{
 		Required:    true,
 		Description: "Specifies the identifier for the database; must be unique for your account. As a best practice for [Database Replication and Failover](https://docs.snowflake.com/en/user-guide/db-replication-intro), it is recommended to give each secondary database the same name as its primary database. This practice supports referencing fully-qualified objects (i.e. '<db>.<schema>.<object>') by other objects in the same database, such as querying a fully-qualified table name in a view. If a secondary database has a different name from the primary database, then these object references would break in the secondary database.",
 	},
+	"drop_public_schema_on_creation": {
```

Comment:
let's use `DiffSuppressFunc: IgnoreAfterCreation,`

Reply:
Yup, used it Kuba's comment